projects/igniteui-angular/src/lib/services/animation/angular-animation-service.ts

```diff
@@ -0,0 +1,17 @@
+import { AnimationBuilder, AnimationReferenceMetadata } from '@angular/animations';
+import { Injectable } from '@angular/core';
+import { IgxAngularAnimationPlayer } from './angular-animation-player';
+import { AnimationService, AnimationPlayer } from './animation';
+
+@Injectable({providedIn: 'root'})
+export class IgxAngularAnimationService implements AnimationService {
+    constructor(private builder: AnimationBuilder) { }
+    public buildAnimation(animation: AnimationReferenceMetadata, element: HTMLElement): AnimationPlayer {
```

Comment:
We should probably call the first argument `animationMetadata` since `animation` refers to an actual built animation that is ready to be played and this is just not it.